target/linux/lantiq/files/arch/mips/boot/dts/lantiq/vr9.dtsi

```diff
@@ -211,6 +211,7 @@
 			compatible = "lantiq,xrx200-pinctrl";
 			#gpio-cells = <2>;
 			gpio-controller;
+			gpio-ranges = <&gpio 0 0 56>;
```

Comment:
Please add an explanation for this change to the commit message, or even make it a separate change, as it "affects" several devices.

Reply:
Good point, we should discuss that briefly: It could also be added to the FRITZ dts, I need it for the 3490 as well. When submitting the 3490 PR, I'll create a shared dts anyway, so that would still be an option. 
Without this definition, the gpio-hog doesn't work, thus I added it for all VR9 devices.

Where do you think does it belong to?

Reply:
I cannot judge that from a technical point of view, but if you apply it for all vr9 devices, you 1. should make that clear and 2. should assess the impact on other devices at least punctually.

Maybe somebody with more substantial expertise on this target can help here.

Reply:
> Without this definition, the gpio-hog doesn't work, thus I added it for all VR9 devices.

Done right for the correct reason. As requested by Adrian, please move this change into an own commit.

Reply:
Thanks, I created a separate commit.